api4/plugin.go

```diff
@@ -10,13 +10,15 @@ import (
 	"io/ioutil"
 	"net/http"
 	"net/url"
+	"time"
 
 	"github.com/mattermost/mattermost-server/mlog"
 	"github.com/mattermost/mattermost-server/model"
 )
 
 const (
 	MAXIMUM_PLUGIN_FILE_SIZE = 50 * 1024 * 1024
+	HTTP_REQUEST_TIMEOUT     = 60 * time.Minute
```

Comment:
Can we rename this to something that describes the specific case, and comment as to the rationale for the value?
```suggestion
    // INSTALL_PLUGIN_FROM_URL_HTTP_REQUEST_TIMEOUT defines a high timeout for installing plugins 
    // from an external URL to avoid slow connections or large plugins from failing to install.
	INSTALL_PLUGIN_FROM_URL_HTTP_REQUEST_TIMEOUT     = 60 * time.Minute
```

Reply:
Done